src/main/java/frc/robot/subsystems/internals/InternalSubsystem.java

```diff
@@ -204,9 +204,21 @@ public void periodic() {
             // Spin the top motor on a timer
             exitTimer.start();
             motorTop.set(0.5);
-            stagingExitBall = true;
```

Comment:
So the way the two ball shot delay was fixed was to remove the final ball state entirely? Perhaps just commenting out the `!stagingExitBall` check on line 166 would be a better solution which still represents the ball count semi-accurately?